version: '3'
services:
  web:
    image: "my-py-app"
    network_mode: host
    volumes:
     - .:/data
     - /etc/localtime:/etc/localtime:ro
     - /etc/timezone:/etc/timezone:ro
    entrypoint: ["python", "/data/run.py"]
    environment:
      - SQLALCHEMY_DATABASE_URI=mysql://jzc:Q110110110@127.0.0.1/rank?charset=utf8mb4
      - PYTHONUNBUFFERED=1
      - SECRET_KEY=sfgwe343
      - REDIS_PASSWD=Q110110110

